File: assembly/src/main/java/de/holisticon/reference/ReferenceApplication.java
package de.holisticon.reference;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import springfox.documentation.swagger2.annotations.EnableSwagger2;

@SpringBootApplication
@EnableSwagger2
public class ReferenceApplication {
    public static void main(final String[] args) {
        SpringApplication.run(ReferenceApplication.class, args);
    }
}

File: assembly/src/main/kotlin/com/ltaake/pound/ReferenceApplication.kt
package com.ltaake.pound

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import springfox.documentation.swagger2.annotations.EnableSwagger2

@SpringBootApplication
@EnableSwagger2
open class ReferenceApplication

fun main(args: Array<String>) {
  SpringApplication.run(ReferenceApplication::class.java, *args)
}
